pkg/operator/ceph/object/objectstore.go

```diff
@@ -1000,3 +999,38 @@ func errorOrIsNotFound(err error, msg string, args ...string) error {
 	}
 	return errors.Wrapf(err, msg, args)
 }
+
+// check whether zone endpoint list need to be updated or not
+func UpdateZoneEndpointList(zones []zoneType, desiredEndpointList []string, zoneName string) bool {
+	updateZoneEndpointList := false
+	isZonePresent := false
+	zoneEndpointsList := []string{}
+	checkZoneEndpoints := make(map[string]bool)
+	if zoneName != "" {
+		if len(zones) > 0 {
+			for _, z := range zones {
```

Comment:
`if len(zones) > 0` is unnecessary. If there are no items in the `zones` list, the for loop won't execute anything.